client/src/app/__tests__/AppSpec.js

```diff
@@ -2114,6 +2114,90 @@ describe('<App>', function() {
 
     });
 
+    describe('bottom-panel', function() {
```

Comment:
Mind our usual layout:

* Two empty lines between tests (`it`), two empty lines between tests + setup or consecutive `describe` blocks.

Reply:
Alright. From what I have seen, this layout behavior is inconsistent throughout the testcases 

cf. 
<img width="800" alt="image" src="https://github.com/user-attachments/assets/f6402df8-3bc5-420a-b092-66132bac02c2">

Reply:
If that is the case, then let's consistently apply new baseline ([boy scout rule](https://eugenhoble.medium.com/basic-programming-principles-the-boy-scout-rule-5283af4e15c0)).